packages/blade/.storybook/react/preview.js

```diff
@@ -12,10 +12,6 @@ export const parameters = {
     'storybook/docs/panel': { index: 0 },
     canvas: { title: 'Stories', index: 1 },
   },
-  // setting default view mode to story when development
-  // otherwise while changing component code storybook resets view to story mode
-  // which hampers productivity
```

Comment:
Is this issue not a concern now?

Reply:
Yeah no. because even on development it will remember the last selected tab.